sync/http/route.go

```diff
@@ -1,130 +1,197 @@
 package http
 
```

Comment:
would it make sense to remove the 'Route' part from the method names ? It s part of the package.

Reply:
the package is name http. What are you referring to?